.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: csharp
 matrix:
   include:
-    - os: osx      
-      sudo: required
-      dotnet: 1.0.4
-    
+    - os: osx
+      osx_image: xcode9      
+      sudo: required      
+      dotnet: 2.0.0    
+before_install:
+  - brew update
 install:
-  - cd build
-  - ulimit -n4096
-  - ./install-dotnet-script.sh  
+  - ulimit -n4096  
```

Comment:
this is missing space, shouldn't it be `ulimit -n 4096`?

Reply:
Has always been `-n4096` since we started building on the Mac. A quick test shows that both works.